app/Http/Controllers/AssetsController.php

```diff
@@ -1261,4 +1261,28 @@ public function auditStore(Request $request, $id)
             return redirect()->to("hardware")->with('success', trans('admin/hardware/message.audit.success'));
         }
     }
+
+    /**
+     * Generate labels for given asset ids.
+     *
+     * @return View
+     */
+    public function generateLabels(Request $request)
```

Comment:
This isn't really different than the normal method that generates labels though? Rather than passing values through the route, I think I'd rather pass the user ID to the method and pull the assets from there, in case the assets have changed from the time the admin has clicked on the user's profile.